databases/week3/Untitled-1.mongodb

```diff
@@ -0,0 +1,185 @@
+    location:'102 Mongodb street',
+    when: new Date("2023-02-25T16:30:00"), 
+    max_reservations: 12, 
+    price:120.00,
+    created_date: new Date('2013-02-21')
+  },
+  {
+    id:3,
+    title: 'Dolmer roller',
+    description: ' A delicious selection of fresh  Dolmer with meat',
+    location:'200 Food street',
+    when: new Date("2023-03-07T17:30:00"), 
+    max_reservations:5, 
+    price:150.00,
+    created_date: new Date('2013-03-01')
+  },
+
+]);
+ 
+
+// insert data in Reservations collection: 
+ db.Reservations.insertMany([
+  {
+    id: 1, 
+    number_of_guests: 4,
+    meal_id:1,
+    created_date: new Date('2023-03-02'),
+    contacr_phonenumber:'1234-3894',
+    contact_name:' Astrid sørsen',
+    contact_email:' astrid@food.com'
+  },
+
+  {
+    id: 2, 
+    number_of_guests: 8,
+    meal_id:3,
+    created_date: new Date('2023-03-01'),
+    contacr_phonenumber:'7777-5555',
+    contact_name:' Lærke Thy',
+    contact_email:' lærke@food.com'
+  },
+
+  {
+   id: 3, 
+    number_of_guests: 5,
+    meal_id:2,
+    created_date: new Date('2023-02-28'),
+    contacr_phonenumber:'7895-3937',
+    contact_name:' Sofie smith',
+    contact_email:' sofie@food.com'
+  },
+ ]); 
+
+
+// insert data in Review collection: 
+db.Review.insertMany([
+  {
+    id:1,
+    title:' Great food',
+    description:' the food was amazing.Highly recommand!',
+    meal_id:3, 
+    stars:5,
+    created_date:new Date('2023-03-07')
+  },
+  {
+    id:2,
+    title:' Tasty food',
+    description:' the food was fresh . I will try it again',
+    meal_id:1, 
+    stars:4,
+    created_date:new Date('2023-03-01')
+  },
+  {
+   id:3,
+    title:' Bad food',
+    description:' the food was bad, and the servece was solw!',
+    meal_id:2, 
+    stars:1,
+    created_date:new Date('2023-03-05')
+  },
+]); 
+
+
+//Get all reviews
+ db.Meal.find();
+ db.Reservations.find();
+ db.Review.find();
+
+ // Add a new review
+
+ db.meal.insertOne({
+   id:4,
+    title: 'Spaghetti',
+    description: ' A delicious Italian Spaghetti ',
+    location:'222 Itatian street',
+    when: new Date("2023-03-03T19:00:00"), 
+    max_reservations: 8, 
+    price:100.00,
+    created_date: new Date('2013-03-01')
+
+ }); 
+ db.Reservations.insertOne({
+  id: 4, 
+    number_of_guests: 2,
+    meal_id:4,
+    created_date: new Date('2023-03-03'),
+    contacr_phonenumber:'5678-1234',
+    contact_name:' Asem Kt',
+    contact_email:' asem@food.com'
+ }); 
+ db.Review.insertOne({
+  id:4,
+    title:' perfect food',
+    description:' the best food i have ever teast!',
+    meal_id:4, 
+    stars:5,
+    created_date:new Date('2023-03-04')
+ });
+
+  // Get a review with any id, fx 1
+  db.Meal.find({id:1}); 
+  db.Reservations.find({name:'Asem Kt'}); 
+  db.Review.find({stars:5});
+
+//Update a review with any id, fx 1. 
+//Update any attribute fx the title or multiple attributes
+db.meal.updateOne(
+  { id: 1 },
+  { $set: { id: 6 } }
+);
+ 
+db.Review.updateMany(
```

Comment:
This query updates all Reviews documents that match your condition stars:5. Did you want that initially?